.gitignore

```diff
@@ -4,3 +4,4 @@ dist
 .vscode
 .DS_Store
 **/*.tsbuildinfo
+.wrangler
```

Comment:
```suggestion
.wrangler

```

Reply:
adding an extra newline 😅 

![Screenshot 2024-10-14 at 14 29 28](https://github.com/user-attachments/assets/e7954658-0c35-4fb9-9f92-660da60feb32)

As I believe that having that is a best practice?
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline